pipelines/csw-test-pipeline.yml

```diff
@@ -161,23 +256,54 @@ jobs:
           cd csw-backend/chalice
           pip install -r requirements-dev.txt
 
-          # Unit test
-          python -m unittest discover -v
-
           cd ../build
           npm install
 
           echo "loading csw enviroment..."
           loadcsw ${PREFIX} ${ENVIRONMENT} /root/.ssh/${ENVIRONMENT}.pub
-          #loadcsw 10.9 ${ENVIRONMENT} /root/.ssh/${ENVIRONMENT}.pub
 
-          echo "deploying csw enviroment..."
-          deploycsw ${ENVIRONMENT}
+          echo "get state..."
+          /usr/local/bin/gulp environment.chalice_s3_retrieve_state --env=${ENVIRONMENT}
+
+          echo "test e2e.."
+          /usr/local/bin/gulp environment.e2e --env=${ENVIRONMENT} --user=dan.jones
+
+    on_success:
+      <<: *success_notify
+
+- name: prod-deploy
+  serial: true
+  plan:
+  - get: csw-backend
+  - get: csw-configuration
+    passed: [uat-deploy]
+    trigger: true
+  - task: csw-prod-deploy-task
+    timeout: 30m
+    config:
+      platform: linux
+      image_resource: *csw_image_resource
+      inputs:
+      - name: csw-backend
+      - name: csw-configuration
+      params:
+        AWS_REGION: 'eu-west-1'
+        ACCOUNT_ID: ((cyber-staging))
+        ENVIRONMENT: 'prod'
+        CD_ROLE: ((cd-role))
+      run:
+        path: sh
+        args:
+        - -eu
+        - -c
+        - |
+
+          echo "deploying to ${ENVIRONMENT}..."
 
     on_success:
       <<: *success_notify
 
-- name: job-csw-test-cleanup
+- name: cleanup
```

Comment:
again should this be uat-cleanup?

Reply:
hope we don't need to do that :) see above comment